assets/src/features/home-page/components/VideoroomHomePage.tsx

```diff
@@ -60,56 +50,67 @@ const VideoroomHomePage: React.FC = () => {
 
   return (
     <HomePageLayout>
-      <section className="flex flex-col items-center gap-y-12 sm:gap-y-18">
+      <section className="flex flex-col items-center gap-y-8 sm:justify-center sm:gap-y-14 2xl:gap-y-20">
```

Comment:
I think this could have even bigger gap between the heading and the rest.
<img width="2560" alt="Screenshot 2023-02-14 at 11 39 05" src="https://user-images.githubusercontent.com/58188916/218712208-eda83249-ab65-4033-95a0-cda5a05ffc88.png">